### DISCUSSION TITLE
Navigation progress displays both spinner and the bar

### DISCUSSION THREAD

**Discussion by alecpl** [livewire/livewire#7024] [7. Bugs (archived)] [answered]
### Livewire version

3.0.6

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

I'm using latest Laravel/Jetstream/Livewire versions.

If I have an `<a>` element with wire:navigate I see both progress bar and a spinner. This does not look right to me. In mobile mode the spinner is above the menu button.

Is it a bug? Is there a way to control that other than using css to hide the spinner?
![spinner](https://github.com/livewire/livewire/assets/546788/8a226fd2-b69c-4b8f-a230-859bfbf2be26)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by binaryfire** [comment]
Did you find a solution to this? Showing both the bar and the spinner looks pretty odd to me. Makes it feel like something's broken.

**Comment by alecpl** [comment]
I'm hiding the circle spinner with:
```css
#nprogress .spinner {
    display: none !important;
}
```

**Comment by f-ricci** [comment]
It would be great if this could be either a path to a partial that we can fully control or at least another entry in the config to have the ability to hide the bar or the spinner individually.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed in https://github.com/livewire/livewire/commit/9c38ae0488447c55bb4e9351c6e9d6401e34fd90 so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
